src/pal/src/exception/machmessage.cpp

```diff
@@ -1087,47 +1107,19 @@ mach_msg_id_t MachMessage::MapNotificationToReplyType(mach_msg_id_t eNotificatio
     case EXCEPTION_RAISE_STATE_IDENTITY_64_MESSAGE_ID:
         return EXCEPTION_RAISE_STATE_IDENTITY_REPLY_64_MESSAGE_ID;
     default:
-        FATAL_ERROR("Unsupported message type: %u", eNotificationType);
-    }
-}
-
-// Fetch the return code from a reply type message.
-kern_return_t MachMessage::GetReturnCode()
-{
-    switch (m_pMessage->header.msgh_id)
-    {
-    case EXCEPTION_RAISE_REPLY_MESSAGE_ID:
-        return m_pMessage->data.raise_reply.ret;
-
-    case EXCEPTION_RAISE_REPLY_64_MESSAGE_ID:
-        return m_pMessage->data.raise_reply_64.ret;
-
-    case EXCEPTION_RAISE_STATE_REPLY_MESSAGE_ID:
-        return m_pMessage->data.raise_state_reply.ret;
-
-    case EXCEPTION_RAISE_STATE_REPLY_64_MESSAGE_ID:
-        return m_pMessage->data.raise_state_reply_64.ret;
-
-    case EXCEPTION_RAISE_STATE_IDENTITY_REPLY_MESSAGE_ID:
-        return m_pMessage->data.raise_state_identity_reply.ret;
-
-    case EXCEPTION_RAISE_STATE_IDENTITY_REPLY_64_MESSAGE_ID:
-        return m_pMessage->data.raise_state_identity_reply_64.ret;
-
-    default:
-        FATAL_ERROR("Unsupported message type: %u", m_pMessage->header.msgh_id);
+        NONPAL_RETAIL_ASSERT("Unsupported message type: %u", eNotificationType);
     }
 }
 
 // Set faulting thread in an exception notification message.
-void MachMessage::SetThread(thread_act_t hThread)
+void MachMessage::SetThread(thread_act_t thread)
 {
-    bool fSet;
+    bool fSet = false;
```

Comment:
The `fSet = false` at line 1142 is not needed now that you've set the fSet to false by default.